clang/lib/AST/Type.cpp

```diff
@@ -1746,10 +1767,15 @@ TagDecl *Type::getAsTagDecl() const {
 
 bool Type::hasAttr(attr::Kind AK) const {
   const Type *Cur = this;
+  // FIXME: MacroQualifiedType is not handled correctly
```

Comment:
What does this mean?

Reply:
Obsolete comment, I fixed it in the lines below. However, I should submit this change upstream.

Reply:
Turns out this change was unnecessary and the problem was caused by something else.